src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -471,8 +565,33 @@ public String getQuestionTypeChoiceOption() {
             errors.add(Const.FeedbackQuestion.MSQ_ERROR_NOT_ENOUGH_CHOICES
                        + Const.FeedbackQuestion.MSQ_MIN_NUM_OF_CHOICES + ".");
         }
+
         //TODO: check that msq options do not repeat. needed?
 
+        // checks for manual options only
+        // TODO: add checks for generated options too
```

Comment:
Will this be done in this PR?

Reply:
It should be done, but I haven't yet figured a way to compute total number of students/teams/instructors inside this method :sweat:

Reply:
Please help me with this @wkurniawan07 , I can't find a way to compute number of students/instructors/teams inside this method. Any suggestions?

Reply:
Other than `msqChoices.size() < maxSelectableChoices`, the other condition checks look like they would similarly apply to generated options. What will `msqChoices.size()` return if the options are auto-generated?

Reply:
If auto-generated `msqChoices.size()` returns 0.

Reply:
What will `numOfMsqChoices` return?

Reply:
It returns 0 too. I found a way to store number of options in `numOfMsqChoices` for generated and non-generated type question. Will push the changes shortly.